chart/kubeapps/README.md

```diff
@@ -491,6 +491,7 @@ Once you have installed Kubeapps follow the [Getting Started Guide](https://gith
 | `kubeappsapis.pluginConfig.core.packages.v1alpha1.versionsInSummary.minor` | Number of minor versions to display in the summary                                        | `3`                      |
 | `kubeappsapis.pluginConfig.core.packages.v1alpha1.versionsInSummary.patch` | Number of patch versions to display in the summary                                        | `3`                      |
 | `kubeappsapis.pluginConfig.core.packages.v1alpha1.timeoutSeconds`          | Value to wait for Kubernetes commands to complete                                         | `300`                    |
+| `kubeappsapis.pluginConfig.kappController.packages.v1alpha1.upgradePolicy` | Default upgrade policy generating version constraints ()                                  | `patch`                  |
```

Comment:
Should we reference this in Carvel documentation created in #4127 ?

Reply:
Absolutely yes, I think so. However, I was waiting to confirm this is the chosen solution before writing about it, that's why I haven't added it yet.

Reply:
Mmm... true, perhaps this behavior is too distant from what an average kubeapps user would expect. Defaulting it back to `none`. Thanks!

BTW, I've added some docs here now the other PR got merged